app/config/index.js

```diff
@@ -15,7 +15,8 @@ const schema = joi.object({
     clearInvalid: joi.bool().default(false),
     strictHeader: joi.bool().default(true)
   }),
-  publicApi: joi.string().default('https://environment.data.gov.uk/arcgis/rest/services/RPA/')
+  publicApi: joi.string().default('https://environment.data.gov.uk/arcgis/rest/services/RPA/'),
+  osMapApiKey: joi.string().default('')
```

Comment:
I think we should allow an empty key with `.allow('')`.  Otherwise no one can run our application if they don't have an OS key locally.

If no key is provided we can just list the parcel details on the views without the maps